Islandora/islandora-starter-site
# Fix errors

# What does this Pull Request do?

Almost nothing! (sorry)
* resolves an error that gets thrown every time you try to index something
* updates the languages config. 



# What's new?

* index the "Search index" view mode of the "Repository Item" website as fulltext, in order to remove the warning: 
`The field Rendered item (rendered_item) on index Default Solr content index is missing view mode configuration for some datasources or bundles. Please review (and re-save) the field settings.`

* I opened and changed nothing and clicked save on the language detection config. It seems they've changed how it was stored since the last time this was done. Therefore this will make future changes clearer.

# How should this be tested?

* Add an object, and see if a "Search API" warning like above shows up
* Save the language detection config and see if there are any changes to config. (hopefully, there aren't)

# Documentation Status

* Does this change existing behaviour that's currently documented? no
* Does this change require new pages or sections of documentation? no
* Who does this need to be documented for? n/a
* Associated documentation pull request(s): ___  or documentation issue ___

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers
